camel/toolkits/ask_news_toolkit.py

```diff
@@ -0,0 +1,380 @@
+
+        This helper method processes the API response and returns the content
+        in the specified format, which could be a string, a dictionary, or
+        both.
+
+        Args:
+            response: The response object returned by the API call.
+            return_type (str): Specifies the format of the return value. It
+            can be "string" to return the response as a string, "dicts" to
+            return it as a dictionary, or "both" to return both formats as a
+            tuple.
+
+        Returns:
+            Union[str, dict, Tuple[str, dict]]: The processed response,
+            formatted according to the return_type argument. If "string",
+            returns the response as a string. If "dicts", returns the response
+            as a dictionary. If "both", returns a tuple containing both
+            formats.
+
+        Raises:
+            ValueError: If the return_type provided is invalid.
+        """
+        if return_type == "string":
+            return response.as_string
+        elif return_type == "dicts":
+            return response.as_dicts
+        elif return_type == "both":
+            return (response.as_string, response.as_dicts)
+        else:
+            raise ValueError(f"Invalid return_type: {return_type}")
+
+    def get_news(
+        self,
+        query: str,
+        n_articles: int = 10,
+        return_type: Literal["string", "dicts", "both"] = "string",
+        method: Literal["nl", "kw"] = "kw",
+    ) -> Union[str, dict, Tuple[str, dict]]:
+        r"""Fetch news or stories based on a user query.
+
+        Args:
+            query (str): The search query for fetching relevant news or
+            stories.
+            n_articles (int): Number of articles to include in the response.
+                (default: :obj:`10`)
+            return_type (Literal["string", "dicts", "both"]): The format of the
+                return value. (default: :obj:`"string"`)
+            method (Literal["nl", "kw"]): The search method, either "nl" for
+                natural language or "kw" for keyword search. (default:
+                :obj:`"kw"`)
+
+        Returns:
+            Union[str, dict, Tuple[str, dict]]: A string, dictionary, or both
+            containing the news or story content, or an error message if the
+            process fails.
+        """
+        try:
+            response = self.asknews_client.news.search_news(
+                query=query,
+                n_articles=n_articles,
+                return_type=return_type,
+                method=method,
+            )
+
+            return self._process_response(response, return_type)
+
+        except Exception as e:
+            raise Exception(
+                f"An error occurred while fetching news for '{query}': {e!s}."
+            )
+
+    def search_reddit(
+        self,
+        keywords: list,
+        return_type: Literal["string", "dicts", "both"] = "string",
```

Comment:
It would be better to align the arguments, like in `get_news` we support `n_articles `, here we can also support `n_threads`